src/api/chat_completions.rs

```diff
@@ -25,7 +25,7 @@ impl Instance {
         &self,
         link_words: Vec<String>,
         avoid_words: Vec<String>,
-    ) -> Result<ClueCollection, Box<dyn std::error::Error>> {
+    ) -> Result<(ClueCollection, Usage), Box<dyn std::error::Error>> {
```

Comment:
Returning a tuple is a little awkward syntax wise. My idea is to put token information under `ClueCollection` as fields